src/pysight/nd_hist_generator/movie.py

```diff
@@ -334,13 +354,8 @@ def __append_flim_data(
 
     def __save_flim_at_once(self):
         for chan in self.channels:
-            np.save(f"{self.outputs['filename'].store.path}_chan_{chan}.npy", self.flim_df[chan])
             z = zarr.open(f'{self.outputs["filename"].store.path}', "r+",)
-            z["Lifetime"][f"Channel {chan}"][...] = np.nanmean(np.squeeze(self.flim_df[chan]), 0)
-            # if len(self.flim_df[chan]) > 1:
-            #     z["Lifetime"][f"Channel {chan}"][...] = np.nanmean(np.squeeze(self.flim_df[chan]), 0)
-            # else:
-            #     z["Lifetime"][f"Channel {chan}"][...] = np.squeeze(self.flim_df[chan])
+            z["Lifetime"][f"Channel {chan}"][...] = self.flim_df[chan]
```

Comment:
Is `flim_df` a DataFrame? Because if it is then it shouldn't be written to a Zarr array, and if it's not then it shouldn't be named such.

Reply:
It's a np.array. I was seeking to make as least changes as possible to this file (and this was the var name already used), but I agree it's really misleading. Will refactor this when we fix the output stuff in general